### PR TITLE
Issue 624 newer older help text

### LINKED ISSUE #624: Improve time option usage
For fd's [time options](https://github.com/sharkdp/fd/blob/master/src/app.rs#L411-L444) consider adding these two tweaks

-  `--newer` and `--older` [visible clap aliases](https://docs.rs/clap/2.32.0/clap/struct.Arg.html#method.visible_aliases)
- default the time component to `00:00:00` when only a date is given

which would turn this verbose incantation

    $ fd -t f --change-newer-than '2020-06-29 00:00:00' abcF08m1002r00ec12843000000a5p /opt/datastore/data/prod/xml/schema/xyz/

into a slightly less verbose and more memorable

    $ fd -t f --newer 2020-06-29 abcF08m1002r00ec12843000000a5p /opt/datastore/data/prod/xml/schma/xyz/

Looks like you're using modified time metadata, so these shorter long names are potentially confusing. But I didn't like `--newermt` and `--oldermt`. Ah, the age old naming challenge.

### PR DESCRIPTION
**Pull request by mitch292**
Only a documentation change so I didn't update the changelog. 

Resolves #624 by adding some additional help text for the new aliases that were originally implemented with PR #645 . 

Also completes the last checkbox in #695 . Let me know if you want me to make any changes!

### CODE REVIEW

**Review by tmccombs — APPROVED** [review]

File: src/app.rs [hunk 1]
```diff
@@ -442,10 +442,12 @@ pub fn build_app() -> App<'static, 'static> {
                 .long_help(
                     "Filter results based on the file modification time. The argument can be provided \
                      as a specific point in time (YYYY-MM-DD HH:MM:SS) or as a duration (10h, 1d, 35min). \
-                     '--change-newer-than' can be used as an alias.\n\
+                     By default, for point in time if no time is specified (HH:MM:SS) it defaults to 00:00:00. \
```
Comment (sharkdp) [inline src/app.rs:445]: I don't really understand that sentence. Maybe simply "If the time is not specified, it defaults to 00:00:00."

File: src/app.rs [hunk 2]
```diff
@@ -442,10 +442,12 @@ pub fn build_app() -> App<'static, 'static> {
...
+                     '--change-newer-than' or '--newer' can be used as an alias.\n\
```
Comment (sharkdp) [inline src/app.rs:446]: "can be used as aliases"?

File: src/app.rs [hunk 3]
```diff
@@ -460,10 +462,11 @@ pub fn build_app() -> App<'static, 'static> {
                 .long_help(
                     "Filter results based on the file modification time. The argument can be provided \
                      as a specific point in time (YYYY-MM-DD HH:MM:SS) or as a duration (10h, 1d, 35min). \
-                     '--change-older-than' can be used as an alias.\n\
+                     '--change-older-than' or 'older' can be used as an alias.\n\
```
Comment (sharkdp) [inline src/app.rs:465]: Should be `--older`

File: src/app.rs (hunk 1 above)
Comment (mitch292) [reply src/app.rs:445]: Yeah I like that better.

File: src/app.rs (hunk 2 above)
Comment (mitch292) [reply src/app.rs:446]: Will change

File: src/app.rs (hunk 3 above)
Comment (mitch292) [reply src/app.rs:465]: Ah sorry about that

**Comment by mitch292** [comment]
Updated the PR!

File: src/app.rs [hunk 4]
```diff
@@ -460,10 +462,11 @@ pub fn build_app() -> App<'static, 'static> {
                 .long_help(
                     "Filter results based on the file modification time. The argument can be provided \
                      as a specific point in time (YYYY-MM-DD HH:MM:SS) or as a duration (10h, 1d, 35min). \
-                     '--change-older-than' can be used as an alias.\n\
+                     '--change-older-than' or '--older' can be used as an alias.\n\
```
Comment (sharkdp) [inline src/app.rs:465]: maybe here also "can be used as aliases", for consistency. Thank you for the updates!

Comment (mitch292) [reply src/app.rs]: Ofc np, just pushed up the changes.

**Review by sharkdp — APPROVED** [review]

**Pull request merged**